setup.py

```diff
@@ -46,6 +46,6 @@
     packages=find_packages(),
     python_requires=">=3.5",
```

Comment:
Could we bump the python minor version up to 3.8?

```suggestion
    python_requires=">=3.8",
```

Reply:
I will meet you midway and upgrade to 3.6, and supply a 3.8 build as well (like I do for DDFacet and KillMS)